<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 226, 213);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_coffee">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 2 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color: rgb(157,52,13);
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>coffee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_softDrink">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 2 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color: rgb(157,52,13);
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>soft drink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_chocolate">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 2 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color: rgb(157,52,13);
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>chocolate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 4 Medium</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>What did you eat/drink today?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_energyDrink">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 2 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color: rgb(157,52,13);
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>energy drink</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tea">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>590</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 2 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color: rgb(157,52,13);
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>tea</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 3 Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Today's caffeine intake</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_today">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 4 Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:limegreen;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>더잠실 3 Regular</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/ 0mg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_back">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/arrow.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource_1.qrc"/>
 </resources>
 <connections/>
</ui>
